docs/source/user-guide/configuration/pip-interoperability.rst

```diff
@@ -0,0 +1,33 @@
+===================================
+Improving interoperability with pip
+===================================
+
+To improve the interoperability between conda and pip,
+conda 4.6.0 added preview support. With this interoperability,
+conda can use pip-installed packages to satisfy dependencies,
+cleanly remove pip-installed software, and replace them with
+conda packages when appropriate.
+
+If you’d like to try the feature, you can set this ``.condarc`` setting::
+
+   conda config --set pip_interop_enabled True
+
```

Comment:
A note informing users that setting `pip_interop_enabled` to True may slow down conda would be a good addition here.